### PR TITLE
Fix typo on Readme

### PR DESCRIPTION
**Pull request by tomas-stefano**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,8 @@ Use `--single-run` if you only want it to run once.
 
 ## Deploying
 
-This can easily be deployed to Heroku but the MAS organisation uses its own deploymen infranstructure. If you're a MAS employee, you can deploy this to QA by trigerring the `cms_commit` pipeline . Refer to the organisational [wiki](https://moneyadviceserviceuk.atlassian.net/wiki/display/TEAMB/Contento+CMS) on how to deploy to our production environment.
+This can easily be deployed to Heroku but the MAS organisation uses its own
+deployment infrastructure. If you're a MAS employee, you can deploy this to QA by trigerring the `cms_commit` pipeline . Refer to the organisational [wiki](https://moneyadviceserviceuk.atlassian.net/wiki/display/TEAMB/Contento+CMS) on how to deploy to our production environment.
```
Comment (benlovell) [inline README.md]: triggering

File: README.md [hunk 2]
```diff
@@ -69,7 +69,8 @@ Use `--single-run` if you only want it to run once.
 
 ## Deploying
 
-This can easily be deployed to Heroku but the MAS organisation uses its own deploymen infranstructure. If you're a MAS employee, you can deploy this to QA by trigerring the `cms_commit` pipeline . Refer to the organisational [wiki](https://moneyadviceserviceuk.atlassian.net/wiki/display/TEAMB/Contento+CMS) on how to deploy to our production environment.
+This can easily be deployed to Heroku but the MAS organisation uses its own
```
Comment (benlovell) [inline README.md]: This got wrapped weirdly somehow.

**Comment by darokel** [comment]
Thanks for correcting this :) :+1:

File: README.md (hunk 2 above)
Comment (tomas-stefano) [inline README.md]: Yes, my vim moves to next line, the chars that pass 80 columns.  Already fixed everything.

**Comment by tomas-stefano** [comment]
@darokel cheers mate :)

**Comment by tomas-stefano** [comment]
@benlovell everything is ok. Can I merge? :D

**Comment by benlovell** [comment]
:ship: :shipit: :rocket:

**Pull request merged**